### ISSUE TITLE
Where is BinanceResponseExcetpion?

### ISSUE THREAD

**Issue by NeutralCode** [sammchardy/python-binance#197]
In client.py, comments says BinanceResponseException is raised,
but exception.py don't have BinanceResponseException.

**Comment by Technorocker** [comment]
you have to import it like this
`from binance.exceptions import BinanceAPIException`
then do a try except like this...
```
		try:
			prices = client.get_ticker()
			for i in prices:
				if i['symbol'] == pair:
					price = float(i['lastPrice'])
					return price

		except e:
			print(e.status_code)
			print(e.message)
```
that will send the error response and the error msg into your log but wont stop your code

**Comment by sammchardy** [comment]
@NeutralCode it should actually be BinanceRequestException, that one is raised if an unexpected response is encountered. Mainly when a non-JSON response is returned.

I'll update the docs to use the correct name.

**Comment by larkolab** [comment]
@sammchardy thank you!

**Comment by sammchardy** [comment]
This has been resolved in v0.6.7
